src/pkg/index.ts

```diff
@@ -0,0 +1,686 @@
+export enum Network {
+  Mainnet,
+  Testnet,
+}
+
+export const createBackend = (networkOrUrl: Network | string) => {
+  if (typeof networkOrUrl === "string") {
+    return new Backend(networkOrUrl);
+  }
+
+  switch (networkOrUrl) {
+    case Network.Mainnet:
+      return new Backend("https://api.yoroi.io");
+    case Network.Testnet:
+      return new Backend("https://api.yoroi.io/testnet");
+    default:
+      throw new Error(`Unknown network: ${networkOrUrl}`);
+  }
+};
+
+export class Backend {
+  constructor(
+    public readonly url: string,
+    public readonly version: string = ""
```

Comment:
I think we should enforce the `v2.1` for now, and then also change the response types to be in `camelCase`